src/renovate.ts

```diff
@@ -101,7 +101,11 @@ class Renovate {
     if (/\s/.test(this.input.token.value)) {
       throw new Error('Token MUST NOT contain whitespace');
     }
+    this.validateConfigFileArgument();
+    this.validateDockerCmdFileArgument();
+  }
 
+  private validateConfigFileArgument(): void {
```

Comment:
```suggestion
  private validateConfigFileArgument(): Promise<void> {
```

Reply:
this doesn't need to be handled in constructor, i already changed that on main branch. the validate function is already async. so it should be no problem to add your async code. 🤗